yarn.lock

```diff
@@ -9267,7 +9267,7 @@ markdown-loader@^2.0.1:
     loader-utils "^1.1.0"
     marked "^0.3.9"
 
-marked@2.1.3, marked@^2.0.0, marked@^2.0.1:
+marked@2.1.3, marked@^2.0.1:
```

Comment:
Maybe we could just depend on `@verdaccio/cli` to avoid this noise in lockfile?

Reply:
Ah, this is only split into nice sub-packages in 6.0 which is still `-next` (https://www.npmjs.com/package/@verdaccio/cli). Maybe someday.

Reply:
> Maybe we could just depend on `@verdaccio/cli` to avoid this noise in lockfile?

Sounds good :+1: Something we can do once they release the final version.